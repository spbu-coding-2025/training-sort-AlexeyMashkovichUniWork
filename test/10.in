1071686388 1068433563 370841547 726351708 -98832527 -910088067 611802787 -965676798 -606774965 994164890 -178604794 -1046807351 -337790142 603341684 -27721042 560458887 -1060801057 -1070877667 780477991 68834545 426418265 -430085789 261414096 -613663937 -776440495 -240082421 968773593 300105283 544647617 589589231 -118545634 542592181 584280969 -821445912 195202064 -588293383 -657792156 -266736974 -480228357 -190825298 -346313909 414908672 -163890826 389637773 -55491469 882129956 -123645165 -42550703 884994112 -416908999 -1047457982 237670552 226747035 287697937 697748438 524048363 -1026126309 592780206 -249588179 592263131 108627612 705608010 61113487 -380833243 957903922 -817426274 104615197 -773630059 -10421424 698128663 109286467 717006490 39295510 1019137464 32902438 1057545864 827525595 982999096 -58746663 638777882 -507651728 -32462822 -197293390 792837131 -818506710 -573286777 243143669 -770891196
