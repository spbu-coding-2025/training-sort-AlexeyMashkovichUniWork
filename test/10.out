-1070877667 -1060801057 -1047457982 -1046807351 -1026126309 -965676798 -910088067 -821445912 -818506710 -817426274 -776440495 -773630059 -770891196 -657792156 -613663937 -606774965 -588293383 -573286777 -507651728 -480228357 -430085789 -416908999 -380833243 -346313909 -337790142 -266736974 -249588179 -240082421 -197293390 -190825298 -178604794 -163890826 -123645165 -118545634 -98832527 -58746663 -55491469 -42550703 -32462822 -27721042 -10421424 32902438 39295510 61113487 68834545 104615197 108627612 109286467 195202064 226747035 237670552 243143669 261414096 287697937 300105283 370841547 389637773 414908672 426418265 524048363 542592181 544647617 560458887 584280969 589589231 592263131 592780206 603341684 611802787 638777882 697748438 698128663 705608010 717006490 726351708 780477991 792837131 827525595 882129956 884994112 957903922 968773593 982999096 994164890 1019137464 1057545864 1068433563 1071686388
